[2022-10-24T20:30:47.116+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T20:12:50.944853+00:00 [queued]>
[2022-10-24T20:30:47.141+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T20:12:50.944853+00:00 [queued]>
[2022-10-24T20:30:47.142+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-24T20:30:47.143+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 3
[2022-10-24T20:30:47.143+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-24T20:30:47.176+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): get_vendas> on 2022-10-24 20:12:50.944853+00:00
[2022-10-24T20:30:47.191+0000] {standard_task_runner.py:54} INFO - Started process 17863 to run task
[2022-10-24T20:30:47.198+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'sales_datawarehouse', 'get_vendas', 'manual__2022-10-24T20:12:50.944853+00:00', '--job-id', '300', '--raw', '--subdir', 'DAGS_FOLDER/sales_dw.py', '--cfg-path', '/tmp/tmpu7dvl14c']
[2022-10-24T20:30:47.201+0000] {standard_task_runner.py:83} INFO - Job 300: Subtask get_vendas
[2022-10-24T20:30:47.204+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/sales_dw.py
[2022-10-24T20:30:48.325+0000] {task_command.py:384} INFO - Running <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T20:12:50.944853+00:00 [running]> on host 83cd0ab98720
[2022-10-24T20:30:48.519+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=pedromenegat
AIRFLOW_CTX_DAG_ID=sales_datawarehouse
AIRFLOW_CTX_TASK_ID=get_vendas
AIRFLOW_CTX_EXECUTION_DATE=2022-10-24T20:12:50.944853+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-24T20:12:50.944853+00:00
[2022-10-24T20:30:48.542+0000] {base.py:71} INFO - Using connection ID 'banco_vendas' for task execution.
[2022-10-24T20:30:50.609+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-10-24T20:30:50.623+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=sales_datawarehouse, task_id=get_vendas, execution_date=20221024T201250, start_date=20221024T203047, end_date=20221024T203050
[2022-10-24T20:30:50.663+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-10-24T20:30:50.716+0000] {local_task_job.py:273} INFO - 2 downstream tasks scheduled from follow-on schedule check
